recipes/SpatialPy/meta.yaml

```
@@ -0,0 +1,52 @@
+
+{% set name = "spatialpy" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ba4dfa018bede9e5ee1d8657898e54b85b7c117d815d47056824a864e603796
+  
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  
+requirements:
+  host:
+    - python >3.5
+    - pip
+    - setuptools >=41.0.1
+    
+  run:
+    - python >3.5
+    - matplotlib >=3.0.3
```

Unless you need a PyQt-based gui, better to depend on `maplotlib-base`:
```suggestion
    - matplotlib-base >=3.0.3
```